### PR TITLE
OCD 3714

### PR DESCRIPTION
**Pull request by mattystank**
PR Includes:

Added Person Icons
Update Spacing & Width 
Replace theme file with code thats within the style guide.

### CODE REVIEW

File: src/app/components/login/login-toggle.jsx [hunk 1]
```diff
@@ -37,14 +60,16 @@ function ChplLoginToggle() {
 
   return (
     <>
+    <ThemeProvider theme={theme}>
```
Comment (andlar) [inline src/app/components/login/login-toggle.jsx]: Are you sure this is required? We don't need a ThemeProvider for every component, it only needs to exist once in a component tree, and having multiple ThemeProviders in the application is already giving us error/warning messages in the console

File: src/app/components/login/login.jsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 import CreateIcon from '@material-ui/icons/Create';
 import SendIcon from '@material-ui/icons/Send';
 import NotInterestedIcon from '@material-ui/icons/NotInterested';
+import theme from '../../themes/theme';
```
Comment (andlar) [inline src/app/components/login/login.jsx]: Why is this added?

File: src/app/themes/theme.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ const theme = createMuiTheme({
     display: 'flex',
   },
   spacing: 4,
+
+   /*Start of CHPL Colorwave*/
```
Comment (andlar) [inline src/app/themes/theme.js]: What is "CHPL Colorwave"? If this is documentation for us as developers we need to know what it is, and I've never heard of it before

File: src/app/themes/theme.js [hunk 4]
```diff
@@ -35,24 +43,22 @@ const theme = createMuiTheme({
     },
     h3: {
       fontSize: '1.750em',
-      fontWeight: 800,
+      fontWeight: 400,
```
Comment (andlar) [inline src/app/themes/theme.js]: Did you make these changes in the code that's going in with the Swagger config stuff? Or is it needed here in this PR? Because if it's already in the other PR it'll possibly conflict here. If it's not needed for this feature it shouldn't be here, as it makes it much harder to tell what's being used and what's not

File: src/app/themes/theme.js [hunk 5]
```diff
@@ -22,7 +24,13 @@ const theme = createMuiTheme({
       dark: '#c2c6ca',
       contrastText: '#000000',
     },
+    error: {
+      main: '#c44f65',
```
Comment (andlar) [inline src/app/themes/theme.js]: Is this color used somewhere in this feature?

File: src/app/themes/theme.js [hunk 6]
```diff
@@ -68,32 +74,40 @@ const theme = createMuiTheme({
...
+/*End of Typography*/
+
+/*Start of Overides */
   overrides: {
-    MuiPaper: {
-      rounded: {
-        borderRadius: '8px',
+    MuiAccordionSummary: {
+      root: {
+        backgroundColor: '#f5f9fd',
```
Comment (andlar) [inline src/app/themes/theme.js]: Are we getting rid of the default slight grey color that's on the accordion headers? That seems like a big change to make in this PR

File: src/app/themes/theme.js [hunk 7]
```diff
@@ -68,32 +74,40 @@ const theme = createMuiTheme({
       fontSize: '0.875em',
     },
   },
+/*End of Typography*/
+
+/*Start of Overides */
```
Comment (andlar) [inline src/app/themes/theme.js]: Is this comment needed? The next line says this is overrides. Comments that don't provide useful information are worse than useless, because they inevitably become obsolete and wrong

**Review by andlar — CHANGES_REQUESTED** [review]
There are a lot of changes here that don't have anything to do with the feature under development. That makes it really hard for anyone reviewing either the code or the feature to make sure they can test all of the places that are touched. Can the changes in the theme be stripped down to just the things that are necessary for this feature?

File: src/app/components/login/login.jsx (hunk 2 above)
Comment (mattystank) [reply src/app/components/login/login.jsx]: Isnt a theme provider & theme need to be provided to have breakpoints? this goes along with your comment above ^^

File: src/app/themes/theme.js (hunk 3 above)
Comment (mattystank) [reply src/app/themes/theme.js]: I change this Color Palette if thats better? Colorwave is define as an arrangement of colors

File: src/app/themes/theme.js (hunk 4 above)
Comment (mattystank) [reply src/app/themes/theme.js]: brought in on the Swagger Ticket

File: src/app/themes/theme.js (hunk 5 above)
Comment (mattystank) [reply src/app/themes/theme.js]: its the color for the error inputs brought in on the Swagger Ticket due to the MUI default red not having enough contrast to be 508 compliant

File: src/app/themes/theme.js (hunk 6 above)
Comment (mattystank) [reply src/app/themes/theme.js]: I can take this out of this PR and discuss the topic next week

File: src/app/components/login/login-toggle.jsx (hunk 1 above)
Comment (mattystank) [reply src/app/components/login/login-toggle.jsx]: > There are a lot of changes here that don't have anything to do with the feature under development. That makes it really hard for anyone reviewing either the code or the feature to make sure they can test all of the places that are touched. Can the changes in the theme be stripped down to just the things that are necessary for this feature?

Ill cut the theme changes down.

File: src/app/components/login/login.jsx (hunk 2 above)
Comment (andlar) [reply src/app/components/login/login.jsx]: This line is added to this file, which creates a variable in the file called `theme`, but that variable is never used. So whether it's needed for breakpoints or whatever, if the variable isn't being used in the file it's not needed in this file

File: src/app/themes/theme.js (hunk 3 above)
Comment (andlar) [reply src/app/themes/theme.js]: If this is an industry standard term then that's probably okay. I googled it upon seeing it and found nothing that suggested that. Also, this is the section of the code called `palette`. Doesn't that, by definition, mean this is the color palette? Why add more words that say almost the same thing?

File: src/app/themes/theme.js (hunk 5 above)
Comment (andlar) [reply src/app/themes/theme.js]: If it's a color needed for a different ticket, then it should be in that ticket

File: src/app/themes/theme.js (hunk 4 above)
Comment (andlar) [reply src/app/themes/theme.js]: If it's not part of this PR then it shouldn't be in this PR

File: src/app/themes/theme.js (hunk 7 above)
Comment (andlar) [reply src/app/themes/theme.js]: Why was this conversation resolved?

**Comment by mattystank** [comment]
Update to PR:

- Revert theme back to code that was only needed for feature.
- Tweak criteria info icon to match old UI
- Alter accordion text for "show details/hide details" rather then "show more/ show less"  
- Accordion text doesn't flip now

Concerns:

1. The muibuttonbase was overriding the makestyles class we had for NestedAccordionLevelOneSummary within criteria.jsx on my local. I'm not sure how to fix this.. Discussion is needed. 

2. ONC & PO wants to add a column within our criteria table to host the info buttons so they align vertically. Again I tired to figure this out by I thought I would do more damage then good if I tired to fix this now.

File: src/app/components/listing/details/criteria/criterion-details-view.jsx [hunk 8]
```diff
@@ -238,11 +246,12 @@ function ChplCriterionDetailsView(props) {
                 <TableCell component="th" scope="row">
                   Test Procedure
                   <ChplTooltip title="The type of test procedure and the version used during testing of the certification criterion functionality.">
-                    <Button>
-                      <InfoOutlinedIcon
+                    <IconButton color="primary">
```
Comment (andlar) [inline src/app/components/listing/details/criteria/criterion-details-view.jsx]: Why does this one have a color on it, but the others don't?

File: src/app/themes/theme.js [hunk 9]
```diff
@@ -6,6 +6,7 @@ const theme = createMuiTheme({
     display: 'flex',
   },
   spacing: 4,
+  /*Start of CHPL Palette */
```
Comment (andlar) [inline src/app/themes/theme.js]: What benefit does this line give? The code is in the CHPL project, and the next line literally says `palette`, so is there anything added here that gives us more information?

**Review by andlar — COMMENTED** [review]
There are changes in this PR that are on files that probably shouldn't be changing; the e2e/files. I don't think this PR should be updating those files
